test/functional/p2p_addr_relay.py

```diff
@@ -75,6 +80,19 @@ def setup_addr_msg(self, num):
         msg.addrs = addrs
         return msg
 
+    def setup_rand_addr_msg(self, num):
+        addrs = []
+        for i in range(num):
+            addr = CAddress()
+            addr.time = self.mocktime + i
+            addr.nServices = NODE_NETWORK | NODE_WITNESS
+            addr.ip = f"{random.randrange(128,169)}.{random.randrange(1,255)}.{1+random.randrange(1,255)}.{1+random.randrange(1,255)}"
```

Comment:
Why the second octet in [1,254] and 3rd and 4th [2,255]?

`1+randrange(1,255)` is the same as `randrange(2,256)`, why the former?

Reply:
A typo, fixed.